src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,426 @@
+= Vert.x for Groovy
+
+If you are using Maven or Gradle, add the following dependency to the _dependencies_ section of your
+project descriptor to access the Vert.x Core API and enable the Groovy support:
+
+* Maven (in your `pom.xml`):
+
+[source,xml,subs="+attributes"]
+----
+<dependency>
+ <groupId>io.vertx</groupId>
+ <artifactId>vertx-core</artifactId>
+ <version>${maven.version}</version>
+</dependency>
+<dependency>
+  <groupId>io.vertx</groupId>
+  <artifactId>vertx-lang-groovy</artifactId>
+  <version>${maven.version}</version>
+</dependency>
+----
+
+* Gradle (in your `build.gradle` file):
+
+[source,groovy,subs="+attributes"]
+----
+compile "io.vertx:vertx-core:${maven.version}"
+compile "io.vertx:vertx-lang-groovy:${maven.version}"
+----
+
+== Verticles
+
+=== Writing Verticles
+
+There are two alternatives to create Verticles in Groovy:
+
+a plain Groovy script
+a Groovy class implementing the {@link io.vertx.core.Verticle} interface or extending the {@link io.vertx.core.AbstractVerticle} class
+
+For example, the next snippet is a valid Groovy verticle:
+
+[source, groovy]
+----
+println "Hello from vertx"
+----
+
+On deployment, by default, Vert.x executes the script. Optionally, your script can provide the `startVertx` and
+`stopVertx` methods. Theses methods are called respectively when the verticle starts and stops:
+
+[source, groovy]
+----
+void vertxStart() {
```

Comment:
I think it would be helpfull to explain if vertxStart() and vertxStop() are sync or async methods

Reply:
ok